GTG/gtk/editor/editor.py

```diff
@@ -30,11 +30,12 @@
 from GTG.core.plugins.api import PluginAPI
 from GTG.core.plugins.engine import PluginEngine
 from GTG.core.task import Task
-from GTG.core.translations import _, ngettext
+# from GTG.core.translations import _, ngettext
 from GTG.gtk.editor import GnomeConfig
 from GTG.gtk.editor.calendar import GTGCalendar
 from GTG.gtk.editor.taskview import TaskView
-from GTG.gtk.help import add_help_shortcut
+# from GTG.gtk.help import add_help_shortcut
+from GTG.gtk.tag_completion import tag_filter
```

Comment:
Can you add todo next to this import saying you want to refactor this into a separate module?